walletnode/api/design/artworks.go

```diff
@@ -307,3 +346,93 @@ var RegisterTaskPayload = Type("RegisterTaskPayload", func() {
 	})
 	Required("taskId")
 })
+
+// SearchArtworkParams are query params to searchArtwork request
+var SearchArtworkParams = func() {
+	Attribute("artist", String, func() {
+		Description("Artist PastelID or special value; mine")
+		MaxLength(256)
+	})
+	Attribute("limit", Int, func() {
+		Description("Number of results to be return")
+		Minimum(10)
+		Default(10)
+		Example(10)
+	})
+
+	Attribute("artist_name", String, func() {
+		Description("Name of the artist")
+		MaxLength(256)
+		Example("Mona Lisa")
+	})
+	Attribute("art", String, func() {
+		Description("Art is artwork title")
+		MaxLength(256)
+	})
+	Attribute("series", String, func() {
+		Description("series refers to artwork series name")
+		MaxLength(256)
+	})
+	Attribute("descr", String, func() {
+		Description("descr refers to artist written statement")
+		MaxLength(256)
+	})
+
+	Attribute("keyword", String, func() {
+		Description("keyword is one of the values in artwork_keyword_set")
```

Comment:
I believe you can write better Description, and with Capitalize first letter

Reply:
@levkoburburas I was trying to write in terms of what query param means..as opposed to what the related field means in a domain entity ..as that is something that has already been written.
but I'll try to improve this.